client/templates/profile/profile.js

```diff
@@ -4,6 +4,10 @@ Template.profile.onCreated(function(){
   DocHead.setTitle(title);
 });
 
+Template.registerHelper('editMode',function(){
```

Comment:
Would you please replace it with the template helper.? avoid use of `registerHelper` helper if it's being used in single template.